cysync/package.json

```diff
@@ -85,6 +85,9 @@
     "@mui/lab": "^5.0.0-alpha.77",
     "@mui/material": "^5.6.1",
     "@mui/styles": "^5.6.1",
+    "@types/pouchdb": "^6.4.0",
```

Comment:
what types are doing in general dependency tree?

Reply:
Nice catch , fixed it